x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/StickyErrorProperties.tsx

```diff
@@ -52,8 +52,7 @@ function TransactionLink({
       path={path}
       query={{
         transactionId: transaction.transaction.id,
-        traceId: transaction.trace.id,
-        banana: 'ok'
+        traceId: transaction.trace.id
```

Comment:
Wat? Thanks for fixing this... No more 🍌

Reply:
:wave: